config/locales/nl_etm.yml

```diff
@@ -316,23 +317,65 @@ nl:
     regional: "Provincies and regio's van Nederlands"
     municipal: "Gemeentes en wijken van Nederlands"
     northern_ireland: "Noord-Ierland"
-    add_description: 'Voeg een beschrijving van je scenario toe...'
     cancel: 'Annuleer'
     uncouple: 'Ontkoppel'
     uncouple_desc: |
-      Weet je zeker dat je het scenario wilt ontkoppelen van externe modellen? 
-      Alle waardes die door externe modellen zijn ingesteld worden hiermee uit 
+      Weet je zeker dat je het scenario wilt ontkoppelen van externe modellen?
+      Alle waardes die door externe modellen zijn ingesteld worden hiermee uit
       het scenario verwijderd en kunnen niet meer worden hersteld. </br></br>
-      Zie voor meer informatie de 
-      <a href="https://docs.energytransitionmodel.com/main/external-coupling#uncoupling-the-scenario" target=\"_blank\"> 
+      Zie voor meer informatie de
+      <a href="https://docs.energytransitionmodel.com/main/external-coupling#uncoupling-the-scenario" target=\"_blank\">
       documentatie</a>.
+    add_description: 'Voeg een beschrijving van je scenario toe...'
     edit_text: 'Pas titel en beschrijving aan'
     save_text: 'Sla veranderingen op'
     cancel_text: 'Annuleren'
     edit_featured_scenario: 'Instellingen voor uitgelicht scenario'
     feature_scenario: 'Maak dit een uitgelicht scenario'
     featured_scenario: 'Uitgelicht scenario'
     created_with: 'Scenario gemaakt met het Energietransitiemodel'
+    users:
+      manage: 'Beheer toegang'
+      manage_description:
+        Hier kun je beheren wie met welke rechten toegang heeft tot je scenario.
+        Klik op de 'Gebruiker uitnodigen' knop om iemand toegang te geven, of gebruik
+        de tabel om gekoppelde gebruikers te beheren.<br /><br />
+        Bekijk alsjeblieft de
+        <a href="https://docs.energytransitionmodel.com/main/#12-saved-scenario-access-management-page" target=\"_blank\">
+        documentatien</a> voor meer informatie en details over gebruikersbeheer, zoals gebruikersrollen.
```

Comment:
Update this now that it is at the bottom of the page @mabijkerk @Charlottevm @kaskranenburgQ